packages/tools/pie-icons-react/package.json

```diff
@@ -58,7 +57,8 @@
     "rollup-plugin-typescript2": "0.34.1"
   },
   "dependencies": {
-    "@babel/preset-react": "7.18.6",
+    "@babel/preset-react": "7.22.5",
+    "@justeattakeaway/pie-icons": "workspace:*",
```

Comment:
I'd avoid referencing 'workspace: *' for `dependencies`.

I suspect you may run into issues resolving outside the monorepo.